src/ring_theory/roots_of_unity.lean

```diff
@@ -319,6 +319,9 @@ end
 lemma pow_ne_one_of_pos_of_lt (h0 : 0 < l) (hl : l < k) : ζ ^ l ≠ 1 :=
 mt (nat.le_of_dvd h0 ∘ h.dvd_of_pow_eq_one _) $ not_le_of_lt hl
 
+lemma ne_one {ζ : M} (hζ : is_primitive_root ζ k) (hk : 1 < k) : ζ ≠ 1 :=
+hζ.pow_ne_one_of_pos_of_lt zero_lt_one hk ∘ (pow_one ζ).trans
```

Comment:
```suggestion
lemma ne_one (hk : 1 < k) : ζ ≠ 1 :=
h.pow_ne_one_of_pos_of_lt zero_lt_one hk ∘ (pow_one ζ).trans
```

Reply:
I've changed the name of the hypothesis for consistence, but the `variable (h : ...)` is not in scope here, so we need to ass it by hand.

Reply:
Isn't it on line 304?

Reply:
At least, I think the change I suggested works locally on my computer.